### ISSUE TITLE
.webform-details-toggle-state fails color contrast

### ISSUE THREAD

**Issue by ericras** [unlcms/project-herbie#1229]
When more than one `<details>` element is present on a webform, an "Expand all" button is added that is styled like a link.  The module hardcodes a blue color that fails color contrast check. 

Depending on a form using `<details>` in conjunction with conditionals (where the elements may not even be visible) this button/link just introduces confusion.  I think it should be hidden entirely.
